cmd/cloud/utils.go

```diff
@@ -0,0 +1,143 @@
+package cloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/safedep/vet/internal/auth"
+	"github.com/safedep/vet/internal/ui"
+	"gopkg.in/yaml.v2"
+)
+
+type TokenResponse struct {
+	AccessToken string `json:"access_token"`
+	ExpireIn    int    `json:"expires_in"`
+	Scope       string `json:"scope"`
+	IDToken     string `json:"id_token"`
+	Tokentype   string `json:"token_type"`
+}
+
+func loadConfiguration() (auth.Config, error) {
```

Comment:
Do not duplicate code. Fundamental rule is not to violate DRY. All of these are part of the `internal/auth` package